frontend/src/hooks/useError.ts

```diff
@@ -1,21 +1,51 @@
-import { useEffect, useContext } from 'react'
-import { AxiosError } from 'axios'
-import ErrorContext from 'contexts/ErrorContext'
+import { useEffect, useContext } from 'react';
+import { AxiosError } from 'axios';
+import ErrorContext from 'contexts/ErrorContext';
 
-export default function useError ({ error, prefix }: {
-  error: AxiosError | undefined,
-  prefix?: string,
+export default function useError({
+  error,
+  prefix,
+}: {
+  error: AxiosError | undefined;
+  prefix?: string;
 }) {
-  const { updateErrorContext } = useContext(ErrorContext)
+  const { updateErrorContext } = useContext(ErrorContext);
   useEffect(() => {
     if (error) {
-      error.message = error.message || 'An unknown error occurred.'
-      if (error.response?.data?.message) {
-        error.message = error.response?.data?.message
+      let message = error.message || 'An unknown error occurred.';
```

Comment:
Do we want undefined error to not show, or do we want `'An unknown error occurred.'`? Might be worth using `let message = error?.message || ...` and dropping the `if (error) { ... }`